ManiVault/src/private/MainWindow.cpp

```diff
@@ -42,6 +42,8 @@
     #define MAIN_WINDOW_VERBOSE
 #endif
 
+#define MAIN_WINDOW_SCREEN_CAPTURE
```

Comment:
I think this prevents me from maximizing the application window since it forces a a fixed windows size `setFixedSize(1280, 720);` further down

Reply:
God catch! Will turn that define off